core/modules/qana/PostPlugin.cc

```diff
@@ -96,6 +100,117 @@ PostPlugin::applyPhysical(QueryPlugin::Plan& plan,
         }
     }
 
+    if (_orderBy) {
+        auto const& selectList = plan.stmtOriginal.getSelectList();
+        auto const& selValExprList = selectList.getValueExprList();
+        std::vector<std::string> validSelectCols;
+        for (auto const& selValExpr : *selValExprList) {
+            LOGS(_log, LOG_LVL_DEBUG, "selValExpr=" << selValExpr);
+
+            std::string alias = selValExpr->getAlias();
+            // If the SELECT column has an alias, the ORDER BY statement must use the alias.
```

Comment:
That could be tricky. The ORDER BY needs to work on the workers when there's a LIMIT clause, and the code that returns the order by clause to the proxy is a bit complicated. Requiring the alias keeps things a lot simpler.